app/src/main/java/ch/hikemate/app/model/route/HikeRoutesRepositoryOverpass.kt

```diff
@@ -254,23 +350,21 @@ class HikeRoutesRepositoryOverpass(val client: OkHttpClient) : HikeRoutesReposit
      * @param memberReader The reader for the member object.
      * @return The list of parsed points for this member.
      */
-    private fun parseMember(memberReader: JsonReader): List<LatLong> {
-      val points = mutableListOf<LatLong>()
+    private fun parseMember(memberReader: JsonReader): List<HikeWay> {
+      val ways = mutableListOf<HikeWay>()
       while (memberReader.hasNext()) {
         val name = memberReader.nextName()
         if (name == "type") {
           val type = memberReader.nextString()
           when (type) {
-            // Lat and Long are in the object, no need to change the reader
-            "node" -> points.add(parseLatLong(memberReader))
```

Comment:
Question: Why do you no longer treat the case of a node?

Reply:
Because node doesn't have any start and end, and doesn't provide anything to our hikes. It's used in OSM to show a POI (like a sign or a water fountain), but this has (currently) nothing to do with our hikes